addons/project_todo/models/res_users.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+
+from odoo import _, api, fields, models, modules
+
+
+class Users(models.Model):
+    _name = 'res.users'
+    _inherit = ['res.users']
+
+    @api.model
+    def systray_get_activities(self):
+        """ Split To-do and Project activities in systray by removing
+            the single project.task activity represented and doing a
+            new query to split them between private/non-private tasks.
+        """
+        activity_groups = super().systray_get_activities()
+        # 1. removing project.task activity group
+        for group in activity_groups:
+            if group.get('model') == 'project.task':
+                activity_groups.remove(group)
+                break
+
+        # 2. creating groups for todo and task seperately
+        query = """SELECT BOOL(t.project_id) as is_task, count(*), act.res_model as model, act.res_id,
```

Comment:
```suggestion
        query = """SELECT BOOL(t.project_id) as is_task, count(*), act.res_model, act.res_id,
```
does it need an alias?